renderer/components/UI/FiatAmountInput.js

```diff
@@ -56,6 +55,7 @@ class FiatAmountInput extends React.Component {
   getRules() {
     return {
       precision: 2,
+      step: '.01',
```

Comment:
should it be `'0.01'` like in other places?

Reply:
Yeah, makes sense.